lib/midi.coffee

```diff
@@ -8,19 +8,36 @@ L = utils = midi = midi_in = ___ = null
   midi = rtMidi
   ___ = utils.____ '[midi]'
 
+# Parse method copied from https://github.com/hhromic/midi-utils-js/blob/master/midiparser.js#L418
```

Comment:
Technically by copying this in Legato now needs to include the MIT license. Probably not a big deal though.